packages/pytest-simcore/src/pytest_simcore/helpers/webserver_workspaces.py

```diff
@@ -0,0 +1,43 @@
+import sqlalchemy as sa
+from aiohttp import web
+from models_library.users import GroupID
+from models_library.workspaces import WorkspaceID
+from simcore_postgres_database.models.workspaces_access_rights import (
+    workspaces_access_rights,
+)
+from simcore_service_webserver.db.plugin import get_database_engine
+from sqlalchemy.dialects.postgresql import insert as pg_insert
+
+
+async def update_or_insert_workspace_group(
+    app: web.Application,
+    workspace_id: WorkspaceID,
+    group_id: GroupID,
+    *,
+    read: bool,
+    write: bool,
+    delete: bool,
+) -> None:
+    async with get_database_engine(app).acquire() as conn:
+        insert_stmt = pg_insert(workspaces_access_rights).values(
+            workspace_id=workspace_id,
+            gid=group_id,
+            read=read,
+            write=write,
+            delete=delete,
+            created=sa.func.now(),
+            modified=sa.func.now(),
```

Comment:
this is not necessary. the DB server does it for you automatically.

Reply:
When I will remove folders db in next PR I can take care of this one + include the triggers if you find them so useful (for me its just painful that I need to add them always manually in the migration)

Reply:
why is it painful? this helps when debugging things to see when these changed.

Reply:
Ok here is proper explanation from above: 

 What I don't like about this approach is that it forces the modified timestamp to change. Usually, the modified timestamp is passed to the frontend (even the default filtering is based on this timestamp for resources such as projects, folders, workspaces, wallets, ...). I believe only the user should be able to change it, and in my case, this is managed through the DB layer in a controlled manner. By the way, it's almost the same thing because the SQLAlchemy function func.now() translated into a pure SQL function, likely the same one used in the trigger.

Consider special situations, such as garbage collection (or similar scenarios where you in theory are doing some additional magic for example via some background tasks), where you need to interact with a DB row. In my opinion, the modified timestamp should not be changed in these cases as it has direct implications for the user listing.